.github/templates/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+*Descriptive Summary of your Bugfix, Feature, or Refactoring.*
+
+## List of Added or Changed Components
+- feature a added to module bar
+- bug b fixed in foo.rs
+
+## Open Questions
+- [ ] should I go with solution A or B?
+
+## Checklist
+- [ ] sensible git history (we are not too strict, but multiple "typo" or "fix" commits are not okay)
```

Comment:
I would phrase this with a specific suggestion since not everyone will be familiar with amending and rebasing, something like:

```
Sensible git history (for example, squash "typo" or "fix" commits). See the [Rewriting History](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History) guide for details.
```